src/app/(sidebar)/explorer/components/TransactionsTable.tsx

```diff
@@ -0,0 +1,114 @@
+import {
+  Badge,
+  Button,
+  CopyText,
+  Icon,
+  Link,
+  Text,
+} from "@stellar/design-system";
+import { rpc as StellarRpc } from "@stellar/stellar-sdk";
+import { DataTable } from "@/components/DataTable";
+import { formatEpochToDate } from "@/helpers/formatEpochToDate";
+import { Fragment } from "react";
+import { DataTableHeader } from "@/types/types";
```

Comment:
Nit: we try to group imports in this order:
- libraries (React, SDS, etc),
- any order for components, helpers, etc. as long as they are grouped together, 
- types
- relative imports from local folder